src/status_im/contexts/wallet/sheets/network_preferences/view.cljs

```diff
@@ -35,14 +38,28 @@
                                                 initial-network-preferences-names
                                                 @network-preferences-names-state))]
     (fn [{:keys [on-save blur? button-label]}]
-      (let [theme            (quo.theme/use-theme)
-            network-details  (rf/sub [:wallet/network-details])
-            mainnet          (first network-details)
-            layer-2-networks (rest network-details)
-            current-networks (filter (fn [network]
-                                       (contains? (get-current-preferences-names)
-                                                  (:network-name network)))
-                                     network-details)]
+      (let [theme                            (quo.theme/use-theme)
+            network-details                  (rf/sub [:wallet/network-details])
+            first-section-networks           (filter (fn [network]
```

Comment:
what is a first-section? same for second-section? can we be more descriptive?

Reply:
perhaps
preferred-section
not-preferred-section 
??

Reply:
unprefered-section

Reply:
The thing is in some places we use both sections with different purposes.

For example here:
<img width="300" src="https://github.com/status-im/status-mobile/assets/18485527/30d8a5e9-04f8-44b7-b147-eaa9cf82581b">

So I tried to pick a more generic naming 🤔

Reply:
Ok thanks for the explanation 🙏